eng/scripts/run_tests.ps1

```diff
@@ -0,0 +1,55 @@
+Param(
+    [string] $serviceDir
+)
+
+$cwd = Get-Location
+
+# 0. Find all test directories
+Write-Host "Finding test directories in 'sdk/$serviceDir'"
+$testDirs = & $PSScriptRoot/get_test_dirs.ps1 -serviceDir sdk/$serviceDir
+# Issues here, not returning any objects
+Write-Host "Found test directories $testDirs"
+
+$runTests = $false
+# 0b. Verify there are test files with tests to run in at least one of these directories
+$testDirs | ForEach-Object {
```

Comment:
Since you're not doing object construction, we try to have a convention using `foreach ()` instead, as it reads a little easier and avoids certain powershell gotchas with pipelining.